### ISSUE TITLE
Suggested accessibility improvements

### ISSUE THREAD

**Issue by martin-nc** [elixir-europe/rdmkit#484]
I'm just writing an accessibility statement for the site and have done a small accessibility audit. There are a few issues that probably need fixing:

- [ ] **Skip links:** if someone is navigating the site with the tab key we need hidden links that will allow them to skip past lists of links, which are tedious to go through with a tab key. So we need hidden links to 'Skip to the content' (and miss the main menu), 'Skip the tools table' (otherwise you have to go through every single link with your tab key!), 'Skip the contributors' (again, loads of links). Skip links should be visible when they become focused using the tab key.
- [x] **Heading levels:**  the headings in the footer are `<h6>`. You can't skip heading levels in HTML. After `<h2>`, for example, you have to have `<h3>`, no matter what it looks like. Some assistive devices rely on the heading hierarchy to navigate. So we need the change the footer headings to `<h2>` and use CSS to make them look smaller. Note: you can skip levels upwards e.g. `<h6>` to `<h2>`, but not the other way round.
- [x] **The button for Training materials** fails the contrast checker.
- [ ] **The links in the life cycle wheel** in the menu and home page don't focus when using tab navigation. You don't know where you are when navigating the wheel with a keyboard. 

For reference:
 - [WAVE accessibility checker ](https://wave.webaim.org/)
 - [UK government accessibility checklist ](https://www.gov.uk/government/publications/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one)
 - [W3C accessibility checklist](https://www.w3.org/WAI/test-evaluate/preliminary/)

And also as a reminder, we are required to meet the 'AA' level of the [The Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/).
